chromium_src/chrome/browser/ui/views/tabs/tab_drag_controller.h

```diff
@@ -20,8 +20,15 @@ using TabDragControllerBrave = TabDragController;
   friend TabDragControllerBrave; \
   virtual void MoveAttached
 #define GetTabGroupForTargetIndex virtual GetTabGroupForTargetIndex
+#define GetAttachedBrowserWidget                   \
```

Comment:
`#define GetAttachedBrowserWidget virtual GetAttachedBrowserWidget`

Reply:
This was a weird thing I encountered when I did like that.

```
roller.h:460:18: error: expected member name or ';' after declaration specifiers
  views::Widget* GetAttachedBrowserWidget();
```

I guess it might be due to `*` but didn't dig further.